db/migrate/20220713014358_create_testimonials.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTestimonials < ActiveRecord::Migration[6.1]
+  def change
+    create_table :testimonials do |t|
+    	t.string :name, null: false
```

Comment:
Acá hay un error de indentación.
```suggestion
   t.string :name, null: false
```